### ISSUE TITLE
How does catalog://ImageNetPretrained/MSRA/R-50 trained?

### ISSUE THREAD

**Issue by MaureenZOU** [facebookresearch/maskrcnn-benchmark#1098]
## ❓ Questions and Help
I am trying to pretrain my own version of backbone, however, I couldn't get it comparable with maskrcnn.

The mew model is trained with mean [102.9801, 115.9465, 122.7717] and std [1.0, 1.0, 1.0] on imagenet and reach Top1-Acc: 76%. However, if I load this pretrained weight to maskrcnn, after training with 40000 iter with batch size 2, it only has 3% mAP, which doesn't make sense. 

It would be really appreciated if anyone can help! Thanks : - )

**Comment by Darshan2701** [comment]
@MaureenZOU Could you please post your config file here?

**Comment by MaureenZOU** [comment]
Issue figure out : ) Thanks! It was due to a bug when loading parameter : _ (

**Comment by hunaodaotianming** [comment]
Hi, I am trying  to replace my own bacbone, but got bad results. How did you initialize the pre-training model?

**Comment by MaureenZOU** [comment]
Actually, the way they code frozen bn is a bit different from our intuitive way. Please check out their frozen bn layer and you will know the problem.

**Comment by hunaodaotianming** [comment]
Thank you for your answer!! But after I changed my  code frozen bn ,my loss still can't go down. I use ShuffleNet as my backbone, and I initialize my pre-training model in train_net.py  with the following code:
    if cfg.MODEL.BACKBONE.CONV_BODY.startswith("S"):
        model_dict = model.state_dict()
        state_dict = torch.load('./pretrained_models/ShuffleNetV2/models/ShuffleNetV2.2.0x.pth.tar')
        new_state_dict = {k:v for k,v in state_dict.items() if k in model_dict}
        model_dict.update(new_state_dict)
        model.load_state_dict(model_dict)
Should I change other codes? Thank you again for your answer!!!

**Comment by MaureenZOU** [comment]
Although I didn't know your code. There are two guesses. 

First, if you use shuffle net, does the feature well fit to the feature pyramid network? 
Second, the default input scale of maskrcnn benchmark is [0,255] - mean. And std is [1,1,1]. Does the shuffle net you load fit to this setting?

Hope this will help.
